cmd/etcd.go

```diff
@@ -150,24 +144,68 @@ func putKeyCommandFunc(cmd *cobra.Command, args []string) {
 
 	reqData, err := json.Marshal(putParameter)
 	if err != nil {
-		cmd.Printf("Failed to put key: %v\n", err)
+		cmd.Println(errors.Wrap(err, "Failed to put key"))
 		return
 	}
 	req, err := getRequest(putPrefix, http.MethodPost, "application/json",
 		bytes.NewBuffer(reqData))
 	if err != nil {
-		cmd.Printf("Failed to put key: %v\n", err)
+		cmd.Println(errors.Wrap(err, "Failed to put key"))
 		return
 	}
 	res, err := dail(req)
 	if err != nil {
-		cmd.Printf("Failed to put key: %v\n", err)
+		cmd.Println(errors.Wrap(err, "Failed to put key"))
 		return
 	}
 
 	cmd.Println(res)
 }
 
+func evictOwnerCommandFunc(cmd *cobra.Command, args []string) {
```

Comment:
We don't use the `args` in this function, so how do we know which DDL we remove?